pkg/status/status.go

```diff
@@ -315,10 +316,11 @@ func (s *status) monitorClusterStatus() {
 			// Create the ClusterOperator in the progressing state if it does not exist
 			// or if it is the first report.
 			if s.clusterOperator == nil {
+				msg := fmt.Sprintf("Progressing towards release version: %s", s.version)
```

Comment:
This should only be set for the `OperatorProgressing` condition. The other conditions should have some other generic message.

Reply:
Okay, perhaps `OperatorAvailable` and `OperatorDegraded` messages could be:
>"Determining status: a new release version is being deployed."

Reply:
Or better yet:
>"Determining status: The Marketplace operator is upgrading to a new release version."

Reply:
In fact, we may just want to use `Determining status` given that it takes a bit of time to determine if the operator is degraded after the new version is rolled out.

Reply:
We may want to change the status to `ConditionUnknown` as well.

Reply:
Why would the `status` be `ConditionUnknown`?

Reply:
Marketplace uses`OperatorDegraded` to convey whether or not it is having an issue. I suggest setting the `OperatorDegraded` status to `ConditionUnknown` given that no tests have been ran to validate that the operator is or isn't encountering issues.

Reply:
We could leave the status of `OperatorAvailable` as `false` given that the new version isn't rolled out yet.

Reply:
Your PR was to ensure the messages were not empty. Lets stick to that in this PR.